project/ReleaseUtils.scala

```diff
@@ -0,0 +1,131 @@
+package build
+
+import java.io.File
+
+import sbt.{Def, Keys, MessageOnlyException}
+import sbt.io.syntax.fileToRichFile
+import sbt.io.IO
+
+/** Utilities that are useful for releasing Bloop */
+object ReleaseUtils {
+
+  /** The path to our installation script */
+  private val installScript = Def.setting { BuildKeys.buildBase.value / "bin" / "install.py" }
+
+  /** The latest commit in the `nailgun` submodule. */
+  private val nailgunCommit = Def.setting {
+    GitUtils.withGit(GitUtils.submoduleGitDir("nailgun").value) { git =>
```

Comment:
A small question: why is this necessary? Can we not reuse sbt dynver's utils for nailgun? `version in NailgunBuild` should give us this commit hash 🤔

Reply:
```scala
scala> (version in build.BuildKeys.NailgunBuild).eval
res0: String = 0.1-SNAPSHOT
```

I didn't manage to get the value I wanted with `gitHeadCommit` either...

Reply:
We can keep it this way and I'll make sbt-dynver work and remove this piece of code in the future.